src/homeworks/homework22/numbers/NumbersTest.java

```diff
@@ -0,0 +1,33 @@
+package homeworks.homework22.numbers;
+
+import java.util.Arrays;
+
+public class NumbersTest {
+    public static void main(String[] args) {
+        ICalc iCalc = (one, two, three) -> (one + two + three) / 3;
+        ICalc iCalc1 = (one, two, three) -> one + two + three;
+        ICalc iCalc2 = (one, two, three) -> {
+            int max = one;
+            if (max < two) max = two;
+            if (max < three) max = three;
+            return max;
+        };
```

Comment:
Rename variables please 
```suggestion
        ICalc sumAndDevisionFunction = (one, two, three) -> (one + two + three) / 3;
```